app/start_local.sh

```
@@ -15,9 +15,12 @@ if ! [ -x "$(command -v docker)" ]; then
 fi
 
 if ! [ -x "$(command -v docker-compose)" ]; then
-    echo 'Error: docker-compose is not installed.' >&2
-    echo 'Please install docker-compose before running this setup script.' >&2
-    exit 1
+    docker compose 2>&1 > /dev/null
```

Shellcheck

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)